src/pages/index.astro

```diff
@@ -1,50 +1,73 @@
 ---
-import HomeLayout from '../layouts/HomeLayout.astro';
+import BaseLayout from '../layouts/BaseLayout.astro';
+import InterledgerIcon from '../components/logos/InterledgerIcon.astro';
+import OpenPaymentsIcon from '../components/logos/OpenPaymentsIcon.astro';
+import RafikiIcon from '../components/logos/RafikiIcon.astro';
 ---
-<HomeLayout title="Developers">
+<BaseLayout title="Developers">
   <main>
     <section class="hero">
       <div class="content-wrapper">
-        <div class="hero-text">
-          <h2>Interledger Developers Portal</h2>
-          <p>Enable seamless exchange of value across payment networks.</p>
-          <div class="hero__cta">
-            <a class="hero__btn" href="/developers/get-started">Read the documentation</a>
-            <a class="hero__outlink" href="https://github.com/interledger">Contribute on Github</a>
-          </div>
+        <div class="text-wrapper">
+          <h1 class="hero__heading heading--2">Interledger Developers Portal</h1>
+          <p class="hero__txt">Help shape the future of interoperable payments and enable seamless exchange of value across payment networks.</p>
+          <a class="hero__btn" href="/developers/get-involved">Get involved</a>
         </div>
-        <div class="hero-image">
+        <div class="image-wrapper">
           <img src="/developers/img/devs.svg" alt="Illustration of developers working at a table" class="hero__img" />
         </div>
       </div>
     </section>
 
+    <section>
+      <div class="content-wrapper">
+        <p class="heading--4">Learn more about the various open-source projects that Interledger is working on.</p>
+      </div>
```

Comment:
we didn't want to include WM here?

Reply:
@AlexLakatos said not. But we can always add it in later if we wanted to.